auth/user.go

```diff
@@ -33,6 +40,12 @@ func (s *UserSession) GetToken(ctx context.Context, conf *oauth2.Config) (*oauth
 	// If session expiry or the AccessTokenExpiry has passed, then reauthenticate with browser login and reassign token.
 	if readErr != nil || cache.SessionExpiry.Before(time.Now()) || cache.AccessTokenExpiry.Before(time.Now()) {
 
+		// This is a configuration option set by WithoutBrowserLogin to provide control over when or if the browser is opened
+		// In the case of no valid current login information is found on the system via ClientID:ClientSecret pairing or a previous valid browser login
```

Comment:
```suggestion
		// When the flag is true and no valid credentials are found on the system via ClientID:ClientSecret pairing or a previous unexpired browser login, the client returns an error instead of automatically opening a browser
```